cluster/docker-cluster/kube-deploy/master.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A scripts to install k8s worker node.
+
+set -e
+
+source ~/docker-cluster/kube-deploy/common.sh
+
+# Start k8s components in containers
+start_k8s(){
+    
+    # Start ectd
+    docker -H unix:///var/run/docker-bootstrap.sock run \
+        --restart=always --net=host -d \
+        gcr.io/google_containers/etcd:$ETCD_VERSION \
+        /usr/local/bin/etcd --addr=127.0.0.1:4001 \
+        --bind-addr=0.0.0.0:4001 \
+        --data-dir=/var/etcd/data
+    
+    # Wait for etcd ready
+    sleep 3
+
+    # We set MASTER_IP here to let config-network know we are deploying a master
+    start-network $MASTER_IP
+
+    clear_old_components
+
+    sed -i "s/VERSION/v${K8S_VERSION}/g" ~/docker-cluster/kube-config/master-multi.json
+
+    if [[ "yes" == $MASTER_CONF ]]; then
+        # Tell kubelet mount user's master configure file inside
+        MASTER_CONF="-v $HOME/docker-cluster/kube-config/master-multi.json:/etc/kubernetes/manifests-multi/master.json"
+    else
+        # Clear any illegal value
+        MASTER_CONF=""
+    fi
+
+    # Start kubelet & proxy, then start master components as pods
+    # TODO for now we did not use SSL/authorization, will be fixed soon
+    docker run --net=host --privileged --restart=always -d $MASTER_CONF \
+        -v /:/rootfs:ro \
+        -v /sys:/sys:ro \
+        -v /dev:/dev \
+        -v /var/lib/docker/:/var/lib/docker:ro \
+        -v /var/lib/kubelet/:/var/lib/kubelet:rw \
+        -v /var/run:/var/run:rw \
+        --net=host \
+        --privileged=true \
+        --name=kube_in_docker_kubelet_$RANDOM \
```

Comment:
I once tried this, but kubelet will try to parse fields in any container start with `k8s` prefix and complain:
```
found a container with the "k8s" prefix, but too few fields (2)
```
So, for now I'm hesitate to fix this in this PR ...